From 67d4c3c7da7ee99a69817fd301186adc3155d538 Mon Sep 17 00:00:00 2001
From: Julien Portalier <julien@portalier.com>
Date: Thu, 30 Oct 2025 15:50:08 +0100
Subject: [PATCH] disable-one-spec

---
 spec/pq/conninfo_spec.cr | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/spec/pq/conninfo_spec.cr b/spec/pq/conninfo_spec.cr
index 5c063da..9a1e261 100644
--- a/spec/pq/conninfo_spec.cr
+++ b/spec/pq/conninfo_spec.cr
@@ -31,6 +31,8 @@ end

 describe PQ::ConnInfo, "parts" do
   it "can have all defaults" do
+    {% if flag?(:windows) %} pending! {% end %}
+
     env_var_bubble do
       ci = PQ::ConnInfo.new
       assert_default_params ci
--
2.34.1
